pkg/build/controller/build/build_controller.go

```diff
@@ -1367,6 +1398,37 @@ func (bc *BuildController) handleBuildConfigError(err error, key interface{}) {
 	bc.buildConfigQueue.Forget(key)
 }
 
+// createBuildTrustedCAConfigMap creates a ConfigMap containing additional trusted
+// certificate authorities used by the build pod.
+// The returned ConfigMap has an owner reference to the provided pod, ensuring proper
+// garbage collection.
+func (bc *BuildController) createBuildTrustedCAConfigMap(build *buildv1.Build, buildPod *corev1.Pod) (*corev1.ConfigMap, error) {
+	configMapSpec := bc.createTrustedCASpec(build)
+	configMapSpec.ObjectMeta.OwnerReferences = append(configMapSpec.ObjectMeta.OwnerReferences, metav1.OwnerReference{
+		APIVersion: "v1",
+		Kind:       "Pod",
+		Name:       buildPod.Name,
+		UID:        buildPod.UID,
+	})
+	configMap, err := bc.configMapClient.ConfigMaps(buildPod.Namespace).Create(configMapSpec)
+	if err != nil {
+		return nil, err
+	}
+	return configMap, nil
+}
+
+// createTrustedCASpec creates a ConfigMap
+func (bc *BuildController) createTrustedCASpec(build *buildv1.Build) *corev1.ConfigMap {
+	return &corev1.ConfigMap{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: buildapihelpers.GetBuildTrustedCAName(build),
+		},
+		Data: map[string]string{
+			"cabundle.crt": bc.buildTrustedCA,
```

Comment:
if this is pem-encoded, then it should be cabundle.pem i think?  not sure on the conventions.

Reply:
.crt and .key are common for pem certs and keys respectively, but tools should not depend on there being a specific extension (they parse based on the PEM headers)

Reply:
This is per @nalind - buildah looks for a .crt file.